test/gtest/common/test_perf.cc

```diff
@@ -196,6 +196,9 @@ test_perf::test_result test_perf::run_multi_threaded(const test_spec &test, unsi
     strncpy(params.uct.tl_name , tl_name.c_str(),  sizeof(params.uct.tl_name));
     params.uct.data_layout = test.data_layout;
     params.uct.fc_window   = UCT_PERF_TEST_MAX_FC_WINDOW;
+    params.msg_size_lst    = NULL;
```

Comment:
i think this should be done here, at least 1-2 test cases. i'm afraid some issues such as memory leaks could be missed if this is not tested at all inside gtest.